src/firebolt/async_db/cursor.py

```diff
@@ -374,34 +424,14 @@ async def _do_execute(
         async_execution: Optional[bool] = False,
     ) -> None:
         self._reset()
+        queries: List[Union[SetParameter, str]] = (
+            [raw_query] if skip_parsing else split_format_sql(raw_query, parameters)
+        )
         try:
-            if (
-                async_execution
-                and self._set_parameters.get("use_standard_sql", "0") == "1"
-            ):
-                raise AsyncExecutionUnavailableError(
-                    "It is not possible to execute queries asynchronously if "
-                    "use_standard_sql is in use."
-                )
-            if parameters and skip_parsing:
-                logger.warning(
-                    "Query formatting parameters are provided with skip_parsing."
-                    " They will be ignored."
-                )
-
-            # Allow users to manually skip parsing for performance improvement
-            queries: List[Union[SetParameter, str]] = (
-                [raw_query] if skip_parsing else split_format_sql(raw_query, parameters)
-            )
-            if len(queries) > 1 and async_execution:
-                raise AsyncExecutionUnavailableError(
-                    "It is not possible to execute multi-statement "
-                    "queries asynchronously."
-                )
             for query in queries:
 
                 start_time = time.time()
-                # our CREATE EXTERNAL TABLE queries currently require credentials,
+                # Our CREATE EXTERNAL TABLE queries currently require credentials,
                 # so we will skip logging those queries.
                 # https://docs.firebolt.io/sql-reference/commands/ddl-commands#create-external-table
```

Comment:
I think it was moved to https://docs.firebolt.io/sql-reference/commands/create-external-table.html

Reply:
Got it.